lib/particle.js

```diff
@@ -0,0 +1,84 @@
+let mcdata
+
+module.exports = loader
+
+function loader (registry) {
+  mcdata = registry
```

Comment:
Global registry variable doesn't make sense. One instance will overwrite the other. You need to put the class inside the loader function.